compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -505,7 +505,7 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
     TABLE_KEY_SPECIFIER_MISMATCH("BCE3301", "table.key.specifier.mismatch"),
     KEY_SPECIFIER_SIZE_MISMATCH_WITH_KEY_CONSTRAINT("BCE3302", "key.specifier.size.mismatch.with.key.constraint"),
     KEY_SPECIFIER_MISMATCH_WITH_KEY_CONSTRAINT("BCE3303", "key.specifier.mismatch.with.key.constraint"),
-    MEMBER_ACCESS_NOT_SUPPORT_FOR_KEYLESS_TABLE("BCE3305", "member.access.not.supported.keyless.table"),
+    MEMBER_ACCESS_NOT_SUPPORT_FOR_TABLE("BCE3305", "member.access.not.supported.table"),
```

Comment:
If the table type is `table<Type> key<never>`; it's a keyless table. Hence, check whether we can differentiate and provide provide detailed error for both cases

Reply:
The error msg is different for the keyless table (`table<Type> key<never>`).  That is `invalid key constraint provided for member access. key constraint expected with type 'never'`